cernopendata/modules/fixtures/data/docs/delphi-about/delphi-about.md

```diff
@@ -0,0 +1,42 @@
+DELPHI was one of four large detectors on the Large Electron-Positron collider (LEP). It took 7 years to design and build, and it started up in 1989. In December 2000, DELPHI stopped taking data and was dismantled to leave room for the construction of the Large Hadron Collider in the LEP tunnel.
+
+## Overview
+DELPHI consisted of a central cylinder filled with subdetectors, with two end-caps. It was 10 meters in length and diameter, and weighed 3500 tons. The detector consisted of 20 subdetectors. A large superconducting magnet sat between an electromagnetic calorimeter (for tracking electrons) and a hadronic calorimeter (to detect hadrons). The magnet generated a field to deflect charged particles so their charge and momenta could be measured.
+
+The collaboration running the detector consisted of about 550 physicists from 56 participating universities and institutes in 22 countries.
+
```

Comment:
What about including a link to the getting-started? I'm not sure if in the `overview`, or maybe in the `documentation`

Reply:
Referencing the opendata site itself ? What will be the URL?

Reply:
It should be `/doc/<slug of the document>', so '/doc/delphi-getting-started`